package.json

```diff
@@ -53,16 +53,16 @@
     "types-ramda": "0.29.4"
   },
   "dependencies": {
-    "@chakra-ui/react": "2.8.2",
-    "@chakra-ui/system": "2.6.2",
+    "@chakra-ui/react": ">=2.8.2",
+    "@chakra-ui/system": ">=2.6.2",
```

Comment:
These are set for the peer dependencies but why are they set for the normal set of dependencies?

Reply:
This is fixed now, but I am not sure why it happened automatically. It took a weird workaround for me to be able to install rollup without having these update.